package/rmt-server.spec

```diff
@@ -247,6 +247,9 @@ if [ $1 -eq 2 ]; then
   fi
 fi
 
+chmod 640 %{conf_dir}/rmt.conf
+chown _rmt %{conf_dir}/rmt.conf
```

Comment:
Couldn't you do the same with `%attr`
See https://stackoverflow.com/a/16951028/3449673 and http://ftp.rpm.org/max-rpm/s1-rpm-specref-files-list-directives.html (or better spend some more time googling this than I did :grin:)

Not an expert here, but this seems cleaner than doing this manually.